### PR TITLE
Use Protobuf Json formatting for routeguide example

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
Two minor issues that will need to be addressed:

* The the compiler directory uses ${protobufVersion}  in its build.gradle.  I am not sure how to get this to work for the examples version
* The route guide data probably needs to be updated for the grpc core project ~eventually~.

### CODE REVIEW

File: examples/build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ dependencies {
             project(':grpc-netty'),
             project(':grpc-protobuf'),
             project(':grpc-stub'),
-            'org.glassfish:javax.json:1.0.4'
+            'com.google.protobuf:protobuf-java-util:3.0.0-beta-2'
```
Comment (ejona86) [inline examples/build.gradle]: Let's put this dep in the 'libraries' map of the root build.gradle and reference it as something like `libraries.protobuf_util` here.

**Comment by ejona86** [comment]
@carl-mastrangelo LGTM, assuming the tweak to how the dependency is specified.

You can create an issue on the grpc repo suggesting they swap to using JSON-encoded Protobuf.

File: examples/build.gradle (hunk 1 above)
Comment (carl-mastrangelo) [inline examples/build.gradle]: Done!

**Comment by carl-mastrangelo** [comment]
Change made and rebuilt.

**Comment by ejona86** [comment]
@carl-mastrangelo LGTM

**Comment by carl-mastrangelo** [comment]
Added followup Issues:

* https://github.com/grpc/grpc/issues/5533
* https://github.com/grpc/grpc-go/issues/583

**Pull request merged**